edx_rest_framework_extensions/auth/jwt/authentication.py

```diff
@@ -77,10 +77,13 @@ def authenticate_credentials(self, payload):
 
 
 def is_jwt_authenticated(request):
-    is_jwt_authenticated = issubclass(
-        type(request.successful_authenticator),
-        BaseJSONWebTokenAuthentication,
-    )
+    is_jwt_authenticated = False
+    successful_authenticator = getattr(request, 'successful_authenticator', None)
+    if successful_authenticator:
+        is_jwt_authenticated = issubclass(
+            type(request.successful_authenticator),
```

Comment:
this only needs to be `type(successful_authenticator),` now that it's set to a var.

Reply:
alternatively, use `if hasattr(request, 'successful_authenticator):`

Reply:
ah, good catch. well update